scripts/run-dev-dependencies.sh

```diff
@@ -7,10 +7,7 @@ set -ex
 function cleanup() {
   docker container stop reverse-proxy
   docker container rm reverse-proxy -v
-  # Same exit code issue for packit stop as packit start....
-  set +e
-  hatch env run -- packit stop ./scripts
-  set -e
+  hatch env run -- packit stop
```

Comment:
I've added support in https://github.com/mrc-ide/packit-deploy/pull/28 to be able to run `hatch env run -- packit unconfigure` here, which would complete cleanup, but we can sort that out in the parent PR perhaps?